src/index.js

```diff
@@ -15,64 +15,120 @@ if (isNaN(pollInterval)) {
 
 
 const metrics = {
-	first_contentful_paint: new Prometheus.Gauge({
-		name: 'pagespeed_first_contentful_paint_milliseconds',
+	desktop_first_contentful_paint: new Prometheus.Gauge({
+		name: 'desktop_pagespeed_first_contentful_paint_milliseconds',
 		help: 'Time to first contentful paint, more info here: https://web.dev/first-contentful-paint/',
 		labelNames: [ 'page' ]
 	}),
-	first_cpu_idle: new Prometheus.Gauge({
-		name: 'pagespeed_first_cpu_idle_milliseconds',
+	desktop_first_cpu_idle: new Prometheus.Gauge({
+		name: 'desktop_pagespeed_first_cpu_idle_milliseconds',
 		help: "First CPU Idle marks the first time at which the page's main thread is quiet enough to handle input.  https://web.dev/first-cpu-idle.",
 		labelNames: [ 'page' ]
 	}),
-	interactive: new Prometheus.Gauge({
-		name: 'pagespeed_interactive_milliseconds',
+	desktop_interactive: new Prometheus.Gauge({
+		name: 'desktop_pagespeed_interactive_milliseconds',
 		help: "Time to interactive is the amount of time it takes for the page to become fully interactive. https://web.dev/interactive.",
 		labelNames: [ 'page' ]
 	}),
-	speed_index: new Prometheus.Gauge({
-		name: 'pagespeed_speed_index_seconds',
+	desktop_speed_index: new Prometheus.Gauge({
+		name: 'desktop_pagespeed_speed_index_seconds',
 		help: "Speed Index, more info here: https://web.dev/speed-index",
 		labelNames: [ 'page' ]
 	}),
-	max_potential_fid: new Prometheus.Gauge({
-		name: 'pagespeed_max_potential_fid_seconds',
+	desktop_max_potential_fid: new Prometheus.Gauge({
+		name: 'desktop_pagespeed_max_potential_fid_seconds',
 		help: "The maximum potential First Input Delay that your users could experience is the duration, in milliseconds, of the longest task. https://developers.google.com/web/updates/2018/05/first-input-delay",
 		labelNames: [ 'page' ]
 	}),
-	first_meaningful_paint: new Prometheus.Gauge({
-		name: 'pagespeed_first_meaningful_paint_seconds',
+	desktop_first_meaningful_paint: new Prometheus.Gauge({
+		name: 'desktop_pagespeed_first_meaningful_paint_seconds',
 		help: "First Meaningful Paint measures when the primary content of a page is visible. https://web.dev/first-meaningful-paint",
 		labelNames: [ 'page' ]
 	}),
-	performance_score: new Prometheus.Gauge({
-		name: 'pagespeed_performance_score',
+	desktop_performance_score: new Prometheus.Gauge({
+		name: 'desktop_pagespeed_performance_score',
 		help: "Performance Score",
 		labelNames: [ 'page' ]
 	}),
-	accessibility_score: new Prometheus.Gauge({
-		name: 'pagespeed_accessibility_score',
+	desktop_accessibility_score: new Prometheus.Gauge({
+		name: 'desktop_pagespeed_accessibility_score',
+		help: "Accessibility Score",
+		labelNames: [ 'page' ]
+	}),
+	mobile_first_contentful_paint: new Prometheus.Gauge({
+		name: 'mobile_pagespeed_first_contentful_paint_milliseconds',
```

Comment:
I'd rename them all like this making them easier to find

```suggestion
		name: 'pagespeed_first_contentful_paint_milliseconds_mobile',
```